pkg/controller/rest/verifiable/operation.go

```diff
@@ -155,7 +161,8 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 	rest.Execute(o.command.GeneratePresentation, rw, req.Body)
 }
 
-// GeneratePresentationByID swagger:route GET /verifiable/credential/{id}/presentation verifiable presentationByIDReq
+// GeneratePresentationByID swagger:route GET /verifiable/credential/{id}/presentation/{did}
```

Comment:
I'm just wondering now that we have DID, should this be POST now with data in the http body rather than path params?

Reply:
all params should be in body + POST

Reply:
yea i wasn't sure about that, but will change this

Reply:
done